daemons/schedulerd/pacemaker-schedulerd.c

```diff
@@ -9,18 +9,18 @@
 
 #include <crm_internal.h>
 
-#include <crm/crm.h>
-#include <stdio.h>
-#include <stdbool.h>
-
-#include <stdlib.h>
-#include <errno.h>
-
-#include <crm/common/cmdline_internal.h>
-#include <crm/common/ipc_internal.h>
-#include <crm/common/mainloop.h>
-#include <crm/pengine/internal.h>
-#include <pacemaker-internal.h>
+#include <signal.h>                     // SIGTERM
+#include <stdbool.h>                    // true
+#include <stddef.h>                     // NULL
+
+#include <glib.h>                       // g_set_error, FALSE, G_OPTION_*
+#include <qb/qblog.h>                   // LOG_INFO, LOG_TRACE
```

Comment:
`LOG_INFO` is a transitive include -- it comes from `syslog.h`, which is included by `qb/qblog.h`. That's fine.